content/cybersecurity/concepts/malware/malware.md

```diff
@@ -0,0 +1,35 @@
+---
+Title: 'Malware'
+Description: 'Malware is malicious software that is designed to invade, damage and disable computer systems, networks, and devices.'
+Subjects:
+  - 'Cybersecurity'
+  - 'Computer Science'
+Tags:
+  - 'Cybersecurity'
+CatalogContent:
+  - 'introduction-to-cybersecurity'
+  - 'paths/fundamentals-of-cybersecurity'
+---
+
+**Malware** is malicious software that is intentionally designed to infiltrate, harm, or disable computer systems, networks, and devices. Depending on the malware's objective, the type of malware can vary, ranging from data to financial theft or sabotaging the system's accessibility.
+
+## Types of Malware
+
+There are different types of malware for different objectives, some of which are:
+
+- A [virus](https://www.codecademy.com/resources/docs/cybersecurity/virus) attaches itself to other programs and replicates once installed.
+- Spyware monitors user activities and reports back to the author.
+- Adware is designed to show advertisements to trick users into installing malicious software.
+- [Ransomware](https://www.codecademy.com/resources/docs/cybersecurity/ransomware) is designed to lock and encrypt files to force the user into paying a ransom to regain access.
+- Trojans, the most dangerous type of malware, can steal financial information or install other types of malware.
+- Worms are similar to viruses. They are self-replicating but have the ability to spread across systems on their own.
+- Keyloggers record user keystrokes, capturing data, and reporting it back to the author.
+- Rootkits provide the author with "root" access while remaining hidden from the user, other software, and the operating system itself.
+
+## Countermeasures to Avoid Malware
+
+- Update the operating system and security software when possible; outdated software may contain [vulnerabilities](https://www.codecademy.com/resources/docs/cybersecurity/vulnerability) which makes it easier for the device to get infected.
+- Never click links that are found on suspicious popups or websites, especially those that offer a free method to paid programs.
+- Never leave a device unattended and unlocked in a public place; applications disguised as spyware can be installed without the user's knowledge.
+- Beware of emails that appear to be from banking institutions requesting to click a link to update the password or regain access to the account. It is recommended to visit the official online banking site and log in from there.
```

Comment:
Not just banking institutions, but any service with which the user might have an account; wireless, Netflix, Microsoft, GMail &c.